pkg/commands/cluster/delete/delete.go

```diff
@@ -4,17 +4,13 @@
 package delete
 
 import (
-	log "github.com/sirupsen/logrus"
 	"github.com/oracle-cne/ocne/pkg/cluster/cache"
 	"github.com/oracle-cne/ocne/pkg/cluster/driver"
 	"github.com/oracle-cne/ocne/pkg/config/types"
+	log "github.com/sirupsen/logrus"
```

Comment:
We should avoid changing the order of imports

Reply:
Thanks! I just changed this.